cmd/machine-config-daemon/start.go

```diff
@@ -81,13 +85,39 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	exitCh := make(chan error)
 	defer close(exitCh)
 
-	var dn *daemon.Daemon
-	var ctx *common.ControllerContext
-
+	var (
+		dn  *daemon.Daemon
+		ctx *common.ControllerContext
+	)
 	glog.Info("starting node writer")
 	nodeWriter := daemon.NewNodeWriter()
 	go nodeWriter.Run(stopCh)
 
+	cb, err := common.NewClientBuilder(startOpts.kubeconfig)
+	if err != nil {
+		if startOpts.onceFrom != "" {
+			glog.Info("Cannot initialize ClientBuilder, likely in onceFrom mode with Ignition")
```

Comment:
"likely"?  We just checked it :wink: 

Why not just

```
if startopts.onceFrom == "" {
   glog.Fatalf(...)
}
```

Reply:
we haven't checked that we're in onceFrom _with_ ignition, that's why, the log is confusing though, need to reword it